packages/scaffolding-cli/templates/build/azDevops/azure/templates/steps/build-node.yml

```diff
@@ -26,6 +26,7 @@ parameters:
   #  Static Code Analysis
   static_code_analysis: false
   sonar_container: ""
+  sonar_env: {}
```

Comment:
`env_vars` for naming consistency?

Reply:
yeah so at this level - this will need to be domain specific - e.g. sonar_env, cypress_env etc... but in the step template itself this will be `env_vars`